bumble/device.py

```diff
@@ -1704,11 +1718,11 @@ async def new_connection():
             asyncio.create_task(new_connection())
 
     @host_event_handler
-    def on_connection_failure(self, connection_handle, error_code):
+    def on_connection_failure(self, transport, connection_handle, peer_address, error_code):
```

Comment:
nit: switch the order of the `transport` and `connection_handle` arguments so that they are in the same order as for the `on_connection` method.